koku/reporting/migrations/0179_matview_tags_hash.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.1.10 on 2021-05-14 18:54
+import logging
+import pkgutil
+
+from django.db import migrations
+
+from reporting.provider.all.openshift.models import VIEWS as OCP_ALL_VIEWS
+
+
+LOG = logging.getLogger(__name__)
+
+
+def add_views(apps, schema_editor):
+    """Create database VIEWS from files."""
+    connection = schema_editor.connection
+    with connection.cursor() as cur:
+        for view in reversed(OCP_ALL_VIEWS):
+            LOG.info(f"""Dropping materialized view "{view}" with cascade""")
+            cur.execute(f"""DROP MATERIALIZED VIEW "{view}" CASCADE;""")
+
+    for view in OCP_ALL_VIEWS:
+        view_sql = pkgutil.get_data("reporting.provider.all.openshift", f"sql/views/{view}.sql")
+        view_sql = view_sql.decode("utf-8")
+        LOG.info(f"""Creating materialized view "{view}"...""")
+        with connection.cursor() as cursor:
+            cursor.execute(view_sql)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("reporting", "0178_auto_20210511_1851")]
```

Comment:
I think the only other thing we need is to update 0178 to make it a no-op so this is the migration that does work.

Reply:
@adberglund Done. See above.